example/app-model/src/main/java/com/mastercard/test/flow/example/app/model/Direct.java

```diff
@@ -124,7 +121,17 @@ public Direct() {
 						rq( HttpReq.PATH, "/count/subset?characters=aeiou" ),
 						rs( deletes ) ) );
 
-		members( flatten( empty, hello, yodel, vowels ) );
+		Flow shouty = Deriver.build( hello, flow -> flow.meta(
+				data -> data.description( "shout" )
+						.motivation( m -> m + ", in a loud environment" ) )
+				.update( UI,
+						rq( ".+", "HELLO WORLD!" ) )
+				.update( CORE,
+						rq( "text", "HELLO WORLD!" ) )
+				.update( HISTOGRAM,
+						rq( ".+", "HELLO WORLD!" ) ) );
+
+		members( flatten( empty, hello, yodel, vowels, shouty ) );
```

Comment:
We're also adding a new flow to make it really obvious that upper-case characters are counted as lower-case.